.travis.yml

```diff
@@ -63,7 +63,8 @@ env:
     - ADOPTOPENJDK=8
     - secure: "P8EqpZoin/YTnwel9TTxSSAHtXfZ4M262BKXlYUZmjoQsjyvXDAeZ7yAqgAvX5BeRFrGkBToPiE+V60stdWkPKs3+9COw2BDUB1CULBHhRY9Lxordmz0xVhgEfsoH4f6r6wOlIQ9kuaWhmP+JdB/mzOHZhLch9ziPi8O46Z8t4k=" # PRIV_KEY_SECRET, for scripts/travis-publish-spec.sh
     - secure: "T1fxtvLTxioyXJYiC/zVYdNYsBOt+0Piw+xE04rB1pzeKahm9+G2mISdcAyqv6/vze9eIJt6jNHHpKX32/Z3Cs1/Ruha4m3k+jblj3S0SbxV6ht2ieJXLT5WoUPFRrU68KXI8wqUadXpjxeJJV53qF2FC4lhfMUsw1IwwMhdaE8=" # PRIVATE_REPO_PASS, for publishing to scala-ci Artifactory
-    - secure: "dbAvl6KEuLwZ0MVQPZihFsPzCdiLbX0EFk3so+hcfEbksrmLQ1tn4X5ZM7Wy1UDR8uN9lxngEwHch7a7lKqpugzmXMew9Wnikr9WBWbJT77Z+XJ/jHI6YuiCRpRo+nvxXGp9Ry80tSIgx5eju0J83IaJL41BWlBkvyAd7YAHORI=" # GPG_SUBKEY_SECRET, to decrypt the pgp key for signing releases
+    - secure: "WImioMUcg+1DYkMbg8ZkAWWmzS4wM6cwfz4eFfoR0QI4R2ENNgAYBVhewhHUz9r+N8Tn0xEfqBOaqV3oCf6qov8vD6ppOn2C2ExaqG290bqYlEEpscYzi7Mfauwt4rePtw3j0vCcC2ch0ktWKopmdG7LkYXpHLXZfLoCbNsJ+Zg=" # GPG_ENC_PASS, to decrypt the gpg key for signing releases
+    - secure: "CwcmYcFw7r6E/gvtdU8/I5Bxt7VmNLCkPhihapxZO/SrJAjDQ5EIiyr+qKeA8sRU8iJ71N8nuENdZLtJvcxpvKNGMyYVNBb9+KT+E/3VwscLEtg9fJLUWfnNL6vgSmxdnHLQbjSOoSrImzN9xMHwEsxHt8MVKbyslDlUnl7aZUw=" # PGP_PASSPHRASE, gpg private key passphrase for signing releases
```

Comment:
sbt-pgp picks up the passphrase from this env var (https://github.com/sbt/sbt-pgp?tab=readme-ov-file#automating-pin-entry-passphrase-entry)